components/collector/tests/base_collectors/test_collector.py

```diff
@@ -192,7 +200,7 @@ async def test_fetch_with_post_error(self):
             await self._fetch_measurements(mock_async_get_request)
         post.assert_called_once_with(
             self.measurement_api_url,
-            json=dict(has_error=False, sources=[self._source()], metric_uuid="metric_uuid"),
+            json=dict(has_error=False, sources=[self._source()], metric_uuid="metric_uuid", report_uuid=""),
```

Comment:
Why doesn't the json contain a report_uuid? Same for all the test cases below without a report_uuid.

Reply:
I didn't add the report_uuid to the fixtures for thse tests since it is not necessary to test what these test tests. 

I do agree however that a testcase should be added to check that if a report_uuid is present, it will be forwarded.

Reply:
Well, you did add a report_uuid, but gave it the value of an empty string. My suggestion is to give it a proper value just like the metric_uuid argument: 
```suggestion
            json=dict(has_error=False, sources=[self._source()], metric_uuid="metric_uuid", report_uuid="report_uuid"),
```

By the way, to prevent SonarQube from complaining about duplicating "metric_uuid" and "report_uuid", there are constants defined in the fixtures module.